superset-installer/etc/reflex-provisioner/playbooks/superset/main.yml

```diff
@@ -9,6 +9,18 @@
     - 'always'
   tasks: []
 
+
+- name: "Set VIP for NONHA setup"
+  hosts:
+    - management-nodes
+  tags:
+    - always
+  tasks:
+    - set_fact:
+        site_floating_ip:
+          ip: "{{ hostvars[groups['management-nodes'][0]]['ansible_default_ipv4']['address'] }}"
+      when: site_deployment_type == 'NONHA'
+
```

Comment:
I mean to say here we should not set `site-floating-ip` instead find its usages.
I suspect it is used to only set `psql-host-ip`
If that is the case then we can create a task to set this value based on deployment type

Reply:
_site-floating-ip_ is primary variable defined in platform playbook and vip  is actually assign by this variable across platform local  variables like  _platform_loadbalancer_vip_01_ and in superset  we  use _platform_loadbalancer_vip_01_ var  to assign _superset_postgres_access_ip_ .

In HA setup __site-floating-ip__ variable is set by platform through loadbalancer-nodes vip but in non HA  loadbalancer-nodes is not defined  so we need to define this variable manually using custom logic. 

Non HA Host config:--
[management-nodes]
platacc001-mst-01.gvs.ggn

[loadbalancer-nodes]

[master-nodes]
platacc001-mst-01.gvs.ggn

[slave-nodes]
platacc001-slv-01.gvs.ggn
platacc001-slv-02.gvs.ggn
platacc001-slv-03.gvs.ggn

Reply:
Exactly my pointment.  Instead of relying on platform-load-balacer-virtual to set `superset_postgres_access_ip`

We should modify this task to set this varible directly using ifelse Based on deployment type

Reply:
This is valid till, we are using Reflex platform. In case of Reflex platform else condition is not needed or unwanted code.

whenever we are going to use customer side platform in that case this task is unwanted that should be removed..

Rest discussion, we can do offline....

Reply:
Sure let's discss offline. Just relock at code block I posted. I am settings a different variable form original code.

Reply:
We are keeping this as-is . Current deployement is tied to Guavus Platform in many ways other than this dependecy